src/sentry/static/sentry/app/views/organizationEventsV2/transactionView/spanBar.tsx

```diff
@@ -0,0 +1,780 @@
+            left: toPercent(dividerPosition),
+            width: toPercent(1 - dividerPosition),
+            backgroundColor: this.state.displayDetail ? '#F0ECF3' : void 0,
+          }}
+        >
+          {displaySpanBar && (
+            <SpanBarRectangle
+              style={{
+                backgroundColor: spanBarColour,
+                left: bounds.left,
+                width: bounds.width,
+              }}
+            />
+          )}
+          <Duration>{durationString}</Duration>
+          {this.renderWarningText({warningText: bounds.warning})}
+        </SpanRowCell>
+        {this.renderDivider(dividerHandlerChildrenProps)}
+      </SpanRowCellContainer>
+    );
+  };
+
+  render() {
+    const bounds = this.getBounds();
+
+    const isSpanVisibleInView = bounds.isSpanVisibleInView;
+
+    return (
+      <SpanRow
+        innerRef={this.spanRowDOMRef}
+        style={{
+          display: isSpanVisibleInView ? 'block' : 'none',
+
+          // TODO: this is a border-top; this needs polishing from a real CSS ninja
+          boxShadow: this.state.displayDetail ? '0 -1px 0 #d1cad8' : void 0,
+        }}
+        onClick={() => {
+          this.toggleDisplayDetail();
+        }}
+      >
+        <DividerHandlerManager.Consumer>
+          {(
+            dividerHandlerChildrenProps: DividerHandlerManager.DividerHandlerManagerChildrenProps
+          ) => {
+            return this.renderHeader(dividerHandlerChildrenProps);
+          }}
+        </DividerHandlerManager.Consumer>
+        {this.renderDetail({isVisible: isSpanVisibleInView})}
+      </SpanRow>
+    );
+  }
+}
+
+const SpanRowCellContainer = styled('div')`
+  position: relative;
+  height: ${SPAN_ROW_HEIGHT}px;
+`;
+
+const SpanRowCell = styled('div')`
+  position: absolute;
+
+  height: ${SPAN_ROW_HEIGHT}px;
+
+  overflow: hidden;
+`;
+
+export const DividerLine = styled('div')`
+  position: absolute;
+  height: ${SPAN_ROW_HEIGHT}px;
+
+  transform: translateX(-50%);
+
+  background-color: #cdc7d5;
+  z-index: 999999;
+
+  &.hovering {
+    width: 4px !important;
+    cursor: col-resize;
+  }
+
+  ${({hovering}: {hovering: boolean}) => {
+    if (!hovering) {
+      return 'width: 2px;';
+    }
+
+    return `
+      width: 4px;
+      cursor: col-resize;
+      `;
+  }};
+`;
+
+const SpanBarTitleContainer = styled('div')`
+  display: flex;
+  align-items: center;
+
+  height: ${SPAN_ROW_HEIGHT}px;
+  position: absolute;
+  left: 0;
+  top: 0;
+  width: 100%;
+`;
+
+const SpanBarTitle = styled('div')`
+  position: relative;
+  top: 0;
+
+  height: ${SPAN_ROW_HEIGHT}px;
+  line-height: ${SPAN_ROW_HEIGHT}px;
+
+  color: #4a3e56;
+  font-size: 12px;
+
+  user-select: none;
+
+  white-space: nowrap;
+
+  display: flex;
+  align-items: center;
+`;
+
+const SpanTreeTogglerContainer = styled('div')`
+  position: relative;
+  top: 0;
+
+  height: 15px;
+
+  max-width: 40px;
+  width: 40px;
+  min-width: 40px;
+
+  margin-right: 8px;
+
+  z-index: 99999;
```

Comment:
Another place we should use `theme.zIndex`.

Reply:
I've decided to collate the z-indices locally https://github.com/getsentry/sentry/pull/14225/commits/3fe6709d60e3d9c17981f6080bca08bcdf55671b